packages/symfony-toggle/composer.json

```diff
@@ -69,7 +69,10 @@
         }
     },
     "suggest": {
-        "doctrine/dbal": "Allows using Dbal toggle management implementation."
+        "pheature/dbal-toggle": "Allows using Dbal toggle management implementation.",
+        "pheature/inmemory-toggle": "Allows using Inmemory toggle management implementation.",
+        "pheature/toggle-crud": "Allows using toggle management CRUD implementation.",
+        "pheature/toggle-crud-psr7-api": "Allows using toggle management CRUD HTTP API."
```

Comment:
The `toggle-crud-psr7-api` will suggest installing `symfony/psr-http-message-bridge` in its composer json file.